### PR TITLE
feat: added responsive to fit TV (16:9 aspect ratio)

### PR DESCRIPTION
**Pull request by johnlhuangP**
I was under the assumption that the TV will have a 16:9 aspect ratio, but please correct me if I am mistaken. I adjusted the sizing of the buttons to fit as the screen enlarges. I did not account for smaller screens.

### CODE REVIEW

File: election-portal/app/assets/finance.svg
Comment (arielshif) [inline election-portal/app/assets/finance.svg:1]: You should check out the discussion under Louis' merge. Icons can get implemented as react-icons. Their size can be adjusted through font-size, which will allow conformity to manual browser scaling.

https://github.com/BPR-Data-Team/Election-Night/pull/97#discussion_r1777620976

File: election-portal/app/page.tsx
Comment (arielshif) [inline election-portal/app/page.tsx:1]: can be easily switched out to react icons. lmk if you want exact code or check out https://github.com/BPR-Data-Team/ElectionModel2024-site/blob/69301156e65214e5a8dc00e489db8168ec71ffa2/src/components/Footer.tsx for an example in prod

**Review by arielshif — CHANGES_REQUESTED** [review]

**Comment by arielshif** [comment]
code looks good! just left a few comments about how we're implementing icons for consistency

**Comment by johnlhuangP** [comment]
Thanks for the comments. I will fix those up.

**Comment by johnlhuangP** [comment]
Hi, just changed to react-icons. I am a little unsure about the icon for Election breakdown because I did not find an icon on react-icons that looked like the one on the figma.

**Review by arielshif — APPROVED** [review]
approved but will let @akslaym merge

**Review by akslaym — APPROVED** [review]

**Comment by akslaym** [comment]
I had this working on a big TV screen which is great.

**Pull request merged**